Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/FindPeakAutomatic.py

```diff
@@ -0,0 +1,395 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+from mantid.simpleapi import (ConvertToPointData, CreateWorkspace, DeleteWorkspace,
+                              CreateEmptyTableWorkspace, FitGaussianPeaks)
+from mantid.api import (DataProcessorAlgorithm, AlgorithmFactory, WorkspaceProperty, Progress, ITableWorkspaceProperty)
+from mantid.kernel import (Direction, FloatBoundedValidator, IntBoundedValidator)
+from mantid import mtd, logger
+
+import numpy as np
+import scipy.signal
+
+
+class FindPeakAutomatic(DataProcessorAlgorithm):
+    _acceptance = 0.01
+    _bad_peak_to_consider = 50
+    _smooth_window = 5
+    _use_poisson_cost = False
+    _fit_to_baseline = False
+    _estimate_peak_sigma = 5
+    _min_sigma = 0.0
+    _max_sigma = 30.0
+    _plot_peaks = None
+    _plot_baseline = None
+
+    def category(self):
+        return 'Optimization\\PeakFinding'
+
+    def summary(self):
+        return 'Locates and estimated parameters for all the peaks in an emission spectra'
```

Comment:
As this is a general algorithm can be avoid specific terms such as `emission`?

Also I think `estimated` -> `estimates`